landscape/client/manager/aptsources.py

```diff
@@ -140,10 +140,15 @@ def _handle_sources(self, ignored, sources):
             "manage_sources_list_d",
             True,
         )
+
         if manage_sources_list_d not in FALSE_VALUES:
-            filenames = glob.glob(os.path.join(self.SOURCES_LIST_D, "*.list"))
-            for filename in filenames:
-                shutil.move(filename, f"{filename}.save")
+            patterns = ["*.list", "*.sources"]
+            for pattern in patterns:
+                filenames = glob.glob(os.path.join(
+                    self.SOURCES_LIST_D, pattern
+                ))
+                for filename in filenames:
+                    shutil.move(filename, f"{filename}.save")
```

Comment:
Not sure if I remember correctly, but I think there's somewhere where these `.save` files get moved back (i.e., have the `.save` removed). If that's true, please make sure that that code also handles `.sources` files.

It might already, but let me know if that's the case.